src/ui/zcl_abapgit_gui_page_repo_over.clas.abap

```diff
@@ -477,83 +477,84 @@ CLASS zcl_abapgit_gui_page_repo_over IMPLEMENTATION.
   METHOD render_actions.
 
     CONSTANTS:
-      lc_separator     TYPE string VALUE `<span class="separator">|</span>`,
       lc_dummy_key     TYPE string VALUE `?key=#`,
       lc_offline_class TYPE string VALUE `action_offline_repo`,
       lc_online_class  TYPE string VALUE `action_online_repo`,
       lc_action_class  TYPE string VALUE `action_link`.
 
-    DATA:
-      lv_settings_link TYPE string,
-      lv_check_link    TYPE string,
-      lv_stage_link    TYPE string,
-      lv_patch_link    TYPE string,
-      lv_diff_link     TYPE string,
-      lv_pull_link     TYPE string.
-
-    DATA:
-      lv_zip_import_link TYPE string,
-      lv_zip_export_link TYPE string.
-
-    ii_html->add( |<div class="float-right">| ).
+    DATA lo_toolbar TYPE REF TO zcl_abapgit_html_toolbar.
```

Comment:
Changing the actions to a proper toolbar is a good idea. However, the actions should be only the frequent used commands. So we should not add the filtered options here unless there's consensus.

Reply:
@mbtools perhaps not everybody uses this button, but I need it on this screen. If I have to open the repository first then it will take a long time until AG has checked everything. Perhaps I could add there also an "Advanced" menu but then it will have just one entry for now...
Do you have a better idea?

Reply:
you could add a keyboard shortcut for it without adding it to the toolbar. or we add a "more" or "..." icon/menu to the end of the toolbar to collect less used options.

Reply:
I moved the new functions to the new "more" sub menu.
![image](https://user-images.githubusercontent.com/59128840/150399619-265508c0-8045-4129-8139-f2c5de4f1519.png)

![image](https://user-images.githubusercontent.com/59128840/150399638-c11b083a-b5a4-4615-ad8f-a09947247bb0.png)